### PR TITLE
Topic recognition

### PR DESCRIPTION
**Pull request by lilCheesePuff**
Name: Morgan Nierfeld 
Student Number: 45307906
This is my code for the Pattern Recognition Problem. It aims to solve problem 4 which is segmenting the ISICs data set with the Improved UNet having a minimum dice coefficient of 0.8 on all layers.

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Sufficiently detailed commit messages
- Code valid for Normal Difficulty
- Test DSC > 0.8

Other Feedback
Really good, nothing to add.

**Comment by rexsw** [comment]
Code looks good for normal difficulty

Tensorflow has functions for shuffling data, make sure to use the libraries you have before adding more to your project

Comments such as 
     #Shuffle the data before the split
      skin_scans, labels = shuffle(skin_scans, labels)
Add nothing to the readability of the code and should be avoided

File: recognition/ImprovedUnetOnISICDataSet/driver.py [hunk 1]
```diff
@@ -0,0 +1,218 @@
+import tensorflow as tf
```
Comment (shakes76) [inline recognition/ImprovedUnetOnISICDataSet/driver.py:1]: Code header block should be added for all scripts/source files

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme OK
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
**Feedback required. Please undo the changes to the Recognition Readme file. I can't merge until this is fixed.**
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.

**Pull request merged**